scripts/get-feature-paths.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Get paths for current feature branch without creating anything
+# Used by commands that need to find existing feature files
+
+set -e
+
+# Source common functions
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
+
+# Get all paths
+eval $(get_feature_paths)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)